CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [1.0.6]
+### Fixed
+- Building when copy PDB files is selected in Build Settings

```

Comment:
You can also include an entry about adding new build option.

Reply:
done